### DISCUSSION TITLE
Invoke utility with parameters

### DISCUSSION THREAD

**Discussion by jdhitsolutions** [mdgrs-mei/PowerShellRun#52] [Ideas]
Another wish list item is a way to pass parameters to utilities. For example, I can get to a command-line utility. 
![prsun-context-menu](https://github.com/user-attachments/assets/a9f49ffa-bc52-43eb-8ddd-7e90d72f2467)

But I can't truly run it because the command requires parameters. I don't think you can detect all possible parameters, but maybe you can provide an intermediate window where the user could enter the parameters or the full command they want to run.

**Comment by mdgrs-mei** [comment]
Another nice idea. I can add the third action `Launch App with Arguments` that opens another prompt for manual arguments.

This is for occasional usage though. If a certain parameter set is frequently used, I would recommend adding function or script block entries.

**Reply by jdhitsolutions** [reply]
All good points for documentation. 😄

**Comment by jdhitsolutions** [comment]
I want to revisit this topic and extend it to scripts and functions. I defined a favorite function, but sometimes, I need to pass a different parameter value.

**Reply by mdgrs-mei** [reply]
Thank you. I'll also revisit this after adding the entry group feature.

**Comment by mdgrs-mei** [comment]
@jdhitsolutions "Launch/Invoke with arguments" action was added by #56 and has been included in the v0.8.0 release.
